### DISCUSSION TITLE
Native Stack Navigator usage

### DISCUSSION THREAD

**Discussion by travisobregon** [react-navigation/react-navigation#9644] [Q&A]
Is `@react-navigation/native-stack` supposed to be used or `react-native-screens/native-stack`?

The docs have `@react-navigation/native-stack`, but [react-native-screens docs](https://github.com/software-mansion/react-native-screens/tree/master/native-stack) use their version

**Comment by kacperkapusciak** [comment]
Hi @travisobregon,
since React-Navigation v6 (which is to be released) `@react-navigation/native-stack` will be the correct way to use native-stack. For v5 use `react-native-screens/native-stack`. The documentation will be updated in the nearest future to clarify this.
Cheers
